src/shared-components/button/style.ts

```diff
@@ -171,7 +171,7 @@ export const baseButtonStyles = ({
 }: BaseButtonStylesTypes) => `
   ${TYPOGRAPHY_STYLE.button(theme)}
   appearance: none;
-  border-radius: ${SPACER.xsmall};
+  border-radius:  ${theme.BORDER_RADIUS.small};
```

Comment:
[Nit] The spacing looks over here--curious that `prettier` didn't fix it

Reply:
done